rpcd/etc/openstack_deploy/user_variables.yml

```diff
@@ -89,3 +89,13 @@ tempest_service_available_ceilometer: False
 neutron_neutron_conf_overrides:
   nova:
     endpoint_type: internal
+
+# Disable glance image cache and reconfigure cinder conversion dir outside of a small root partition
+glance_flavor: keystone
+glance_glance_api_conf_overrides:
+  DEFAULT:
+    image_cache_dir: None
```

Comment:
That's not an issue since the cache would not be used, no matter if the file is there. Currently there is no way of suppressing the full installation of glance-image-cache and in my eyes it's not so important to implement that.